swagger/v1/laa_reference.json

```diff
@@ -25,9 +25,12 @@
     },
     "user_name": {
       "example": "example-username",
```

Comment:
can you change to something with less than 10 characters? eg "foobar". As the current example is larger than this, it is failing the checks